lntest/node/harness_node.go

```diff
@@ -876,6 +876,33 @@ func (hn *HarnessNode) RestoreDB() error {
 	return nil
 }
 
+// UpdateGlobalPolicy updates a node's global channel policy and returns the
+// expected policy for further check. It changes the node's `FeeBaseMsat` from
+// 1000 msat to 500,000 msat, and `FeeProportionalMillonths` from 1 msat to 1000
+// msat.
+func (hn *HarnessNode) UpdateGlobalPolicy(baseFeeMsat int64, feeRate float64,
```

Comment:
This seems an odd choice as far as composability goes, shouldn't we pass in the `lnrpc.RoutingPolicy`? rather than making this function a constructor *and* a setter?

Reply:
yes totally, I didn't take a closer look when I was migrating this method. Now simplified it to a minimum I think.